src/generateTasks.js

```diff
@@ -1,26 +1,20 @@
 'use strict'
 
+const path = require('path')
 const minimatch = require('minimatch')
-
-const readConfigOption = require('./readConfigOption')
+const getConfig = require('./getConfig')
+const resolveGitDir = require('./resolveGitDir')
 
 module.exports = function generateTasks(config, files) {
-  const linters = config.linters !== undefined ? config.linters : config
-  const resolve = file => files[file]
+  const { linters, gitDir, globOptions } = getConfig(config) // Ensure we have a normalized config
```

Comment:
this is incompatible with node 4, right?

Reply:
I mean, the import syntax

Reply:
Correct.

Reply:
Oh that sucks. I will need to update eslint node plugin settings...

Reply:
I don't see an `engines` field in package.json.

Reply:
Yeah, I was under impression I already have it in this project but I don't. See https://github.com/okonet/lint-staged/pull/243